docs/types/value-types.rst

```diff
@@ -511,9 +513,15 @@ character sequence ``abcdef``.
     "\n\"\'\\abc\
     def"
 
-Any unicode line terminator which is not a newline (i.e. LF, VF, FF, CR, NEL, LS, PS) is considered to
+Any Unicode line terminator which is not a newline (i.e. LF, VF, FF, CR, NEL, LS, PS) is considered to
 terminate the string literal. Newline only terminates the string literal if it is not preceded by a ``\``.
 
+Unicode Literals
+----------------
+
+While regular string literals can only contain ASCII, Unicode literals prefixed with the keyword ``unicode`` can contain any valid UTF-8 sequence.
+They also support the very same escape sequences as regular string literals.
+
```

Comment:
Can you add example code?

Reply:
Or at least a snippet that explain what "prefixed with the keyword ``unicode``" means.